test/com/facebook/buck/android/AndroidPrebuiltAarIntegrationTest.java

```diff
@@ -16,6 +16,10 @@
 
 package com.facebook.buck.android;
 
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
```

Comment:
You don't need to fix this unless @natthu has any suggested changes, but this is unused.